themes/bootstrap3/templates/layout/js-configs.phtml

```diff
@@ -0,0 +1,7 @@
+<?php /* Global context config only */
+
+$this->jsConfigs()->addConfigPaths(
+    []
```

Comment:
I realize this is probably here as a placeholder for when we add actual configurations... but should we perhaps include a commented-out example in the meantime, or else remove it until it is needed?